_benchmark/reference/commands/run.md

```diff
@@ -142,21 +143,21 @@ The following options help customize how OpenSearch Benchmark provisions OpenSea
 
 ### Workload
 
-The following options determine which workload is used to run the test:
+The following options determine which workload is used during the run:
 
 - `--workload-repository`: Defines the repository from which OpenSearch Benchmark loads workloads. 
 - `--workload-path`: Defines the path to a downloaded or custom workload.
 - `--workload-revision`: Defines a specific revision from the workload source tree that OpenSearch Benchmark should use.
 - `--workload`: Defines the workload to use based on the workload's name. You can find a list of preloaded workloads using `opensearch-benchmark list workloads`.
 
-### Test procedures
+### Run procedures
 
-The following options define what test procedures the test uses and which operations are contained inside the procedure: 
+The following options define what run procedures the run uses and which operations are contained inside the procedure: 
 
-- `--test-execution-id`: Defines a unique ID for this test run.
-Defines the test procedures to use with each workload. You can find a list of test procedures that the workload supports by specifying the workload in the `info` command, for example, `opensearch-benchmark info --workload=<workload_name>`. To look up information on a specific test procedure, use the command `opensearch-benchmark info --workload=<workload_name> --test-procedure=<test-procedure>`.
-- `--include-tasks`: Defines a comma-separated list of test procedure tasks to run. By default, all tasks listed in a test procedure array are run.
-- `--exclude-tasks`: Defines a comma-separated list of test procedure tasks not to run.
+- `--test-run-id`: Defines a unique ID for this run.
```

Comment:
Should we simplify this to `test-id` instead of `test-run-id`? @gkamat We haven't made this change yet so it's possible to switch to `test-id` to keep it simple...